### PR TITLE
Prettify map & add place function

### PR DESCRIPTION
**Pull request by Kauhsa**
- Animated button panel for adding a place
- Animated button panel style changed
- "Add place" circle replaced to middle of shown map
- "Add place" circle style changed
- Place circle style changed
- Some refactoring

### CODE REVIEW

**Comment by Kauhsa** [comment]
Some of the changes (on 4.4):

new_thing

File: App/src/main/java/cc/softwarefactory/lokki/android/fragments/MapViewFragment.java [hunk 1]
```diff
@@ -259,47 +262,47 @@ public void setAddPlacesVisible(boolean visible) {
             ((ImageView) getView().findViewById(R.id.addPlaceCircle)).setImageDrawable(null);
```
Comment (josalmi) [inline App/src/main/java/cc/softwarefactory/lokki/android/fragments/MapViewFragment.java]: Could this go to the else block instead

File: App/src/main/res/anim/add_place_buttons_hide.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android" android:shareInterpolator="false">
+    <translate
+        android:fromXDelta="0%"
+        android:toXDelta="0%"
+        android:fromYDelta="0%"
+        android:toYDelta="100%"
+        android:duration="300" />
+</set>
```
Comment (josalmi) [inline App/src/main/res/anim/add_place_buttons_hide.xml]: No new line at end of file

File: App/src/main/res/anim/add_place_buttons_show.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android" android:shareInterpolator="false">
+    <translate
+        android:fromXDelta="0%"
+        android:toXDelta="0%"
+        android:fromYDelta="100%"
+        android:toYDelta="0%"
+        android:duration="300" />
+</set>
```
Comment (josalmi) [inline App/src/main/res/anim/add_place_buttons_show.xml]: No new line at end of file

File: App/src/main/res/drawable/bottom_panel.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        <shape android:shape="rectangle" >
+            <solid android:color="#1e000000" />
+        </shape>
+    </item>
+    <item android:top="1dp" >
+        <shape android:shape="rectangle" >
+            <solid android:color="#fff" />
+        </shape>
+    </item>
+</layer-list>
```
Comment (josalmi) [inline App/src/main/res/drawable/bottom_panel.xml]: No new line at end of file

File: App/src/main/res/drawable/bottom_panel.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item>
+        <shape android:shape="rectangle" >
+            <solid android:color="#1e000000" />
```
Comment (josalmi) [inline App/src/main/res/drawable/bottom_panel.xml]: Use colors from colors.xml?

File: App/src/main/res/drawable/bottom_panel.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+        </shape>
+    </item>
+    <item android:top="1dp" >
+        <shape android:shape="rectangle" >
+            <solid android:color="#fff" />
```
Comment (josalmi) [inline App/src/main/res/drawable/bottom_panel.xml]: Use white

**Pull request merged**